.konflux/lock-runtime/redhat.repo

```diff
@@ -9,8808 +9,9228 @@
 # a "yum repolist" to refresh available repos
 #
 
-[satellite-client-6-for-rhel-9-x86_64-eus-debug-rpms]
-name = Red Hat Satellite Client 6 for RHEL 9 x86_64 - Extended Update Support (Debug RPMs)
```

Comment:
having such a large repo file is redundant and makes it hard to review the important changes. let's please include only the enabled repos and further filter the used repos according to the generated lockfile.

Reply:
This is straight copied out of the container used to generate the lockfile, is there any simple way to filter it down?